test/models/test_video_vqvae.py

```diff
@@ -87,6 +87,25 @@ def encoder(self, params):
         enc.eval()
         return enc
 
+    @pytest.fixture
+    def downsampler(self, params):
+        in_channel_dims, _, kernel_sizes, _ = params
+        strides = ((2, 2, 2), (2, 2, 2))
+        model = VideoEncoder(
+            in_channel_dims=in_channel_dims,
+            kernel_sizes=kernel_sizes,
+            strides=strides,
+            output_dim=2,
+            n_res_layers=1,
+            attn_hidden_dim=2,
+        )
+        model.eval()
+        return model
+
+    @pytest.fixture
+    def big_input(self):
+        return torch.ones(1, 2, 8, 8, 8)
```

Comment:
It's typical for video shape to be different along the dims.
Use uneven values for each dim, e.g., `(1, 2, 8, 64, 64)`?